WooCommerce/Classes/ViewRelated/Orders/Simple Payments/Amount/SimplePaymentsAmount.swift

```diff
@@ -77,7 +79,19 @@ extension SimplePaymentsAmountHostingController: UIAdaptivePresentationControlle
     }
 
     func presentationControllerShouldDismiss(_ presentationController: UIPresentationController) -> Bool {
-        !rootView.viewModel.disableViewActions
+        guard ServiceLocator.featureFlagService.isFeatureFlagEnabled(.splitViewInOrdersTab) else {
+            return !rootView.viewModel.disableViewActions
```

Comment:
This condition is needed to prevent the view to be dismissed (or the cancel confirmation to be shown) while a network request is being made.

So I think we need it even if `. splitViewInOrdersTab` is enabled.

Reply:
I think this might be redundant since we're already checking if there's an amount entered - and it's highly likely that we only have a network request made with an amount. But I updated the check for `shouldEnableSwipeToDismiss` in b6469212b5854a5c7c045e1bae3235a70bb139ef for extra safety.